ui/src/views/Settings.vue

```diff
@@ -456,7 +456,11 @@ export default {
                 address.time.charAt(address.time.length - 1) == "*"
                   ? "every"
                   : address.time.charAt(address.time.length - 1),
-              time: address.time.charAt(2)
+              time: 
+                address.time.charAt(3).trim() == ""
```

Comment:
change with:
```js
time: address.time.split(" ")[1]
```

It should fix the problem

Reply:
Yes, you are right. Changed.